Source/WebCore/dom/Element.idl

```diff
@@ -60,6 +60,7 @@
 
     ShadowRoot attachShadow(ShadowRootInit init);
     [ImplementedAs=shadowRootForBindings, CallWith=CurrentGlobalObject] readonly attribute ShadowRoot? shadowRoot;
+    [EnabledForWorld=shadowRootIsAlwaysOpen] readonly attribute ShadowRoot? openOrClosedShadowRoot;
```

Comment:
I don't think we want to expose this to every world which forces shadow root be always open.
e.g. DOM wrapper world Safari uses to implement autofill has no business having access to this API.